CurranSeam/Home-Surveillance-System
# Remove all .pyc files and add *.pyc to .gitignore

# Summary
We need to ignore .pyc files since these are compiled python files and only make it difficult to review PRs. Also, these aren't needed since they are generated when the app is executed. 

## What I did here
- Remove all .pyc files from repo and added *.pyc to .gitignore.

## How to test
- Run app using this branch and run `git status`. If *.pyc doesn't appear, we are good to go!
